content/guides/mobile/android/index.md

```diff
@@ -0,0 +1,11 @@
+---
+related_pages:
+ - sdks-and-tools
+---
+
+# Android
+
+The [Box Android SDK][android-sdk] provides native access to the Box API from
+within your Android project.
+
```

Comment:
Definitely needs some more content. Ideas:

* Image to show what you can do?
* Maybe an overview of what the SDK enables (and doesn't enable?)

Reply:
Added image + additional text